packages/botonic-react/src/util/environment.js

```diff
@@ -0,0 +1,27 @@
+export const isDev = process.env.NODE_ENV === 'development'
+export const isProd = process.env.NODE_ENV === 'production'
+
+export const staticAsset = path => {
+  try {
+    const scriptBaseURL = document
+      .querySelector('script[src$="webchat.botonic.js"]')
+      .getAttribute('src')
+    const scriptName = scriptBaseURL.split('/').pop()
+    const basePath = scriptBaseURL.replace('/' + scriptName, '/')
+    return basePath + path
+  } catch (e) {
+    return path
+  }
+}
+
+export const resolveImage = src => {
+  if (isURL(src)) return src
+  return staticAsset(src)
+}
+
+export const isURL = urlPath => {
+  // @stephenhay (38 chars) from: https://mathiasbynens.be/demo/url-regex
+  const URL_PATTERN = /^(https?|ftp):\/\/[^\s/$.?#].[^\s]*$/
```

Comment:
imho I only define local constants when their name helps us to understand what they do. here it's obvious